harness/tests/experiment/fixtures/estimator_no_op/model_def.py

```diff
@@ -27,7 +26,6 @@ def __init__(self, ctx):
 
     def on_trial_close(self) -> None:
         if self.ctx.distributed.get_rank() == 0:
-            time.sleep(15)
```

Comment:
I don't mind dropping this sleep, but it was for a specific regression.  If we feel it's no longer necessary, then probably there's a test built around this that is also unnecessary.

The regression was that we were killing all workers as soon as any worker said it was done.  (see commit `1c8328d07e482345e6d83325d2e9b404207ee873`).

But that was several refactors ago, this _really_ shouldn't be a risk anymore.  Our termination sequence doesn't resemble the old one.

So yes, please also remove `test_on_trial_close_callback`.

This sleep was intentional actually.

Gotta remember how it was failing before... not sure if it's still a regression test worth having.

Reply:
Removed. Didn't know about the workers thing, but the sleep was just making this test run for ~45 seconds, which seems excessive.